static/css/partials/_theme.scss

```diff
@@ -27,6 +27,75 @@
   }
 }
 
+// placeholder for download a/b test of notification bar
+// TODO: use photon colors or existing site style colors instead?
+$test-teal: #00a0c0;
+$test-link-blue: #0297f8;
+$test-yellow: #ffe900; // photon design system: yellow 50
+$test-blue: #0099f6;
+$test-lt-grey: #efeff1;
+$test-dk-grey: #37383d;
```

Comment:
@johngruen Are these colors ok to use? Used a color picker to pull them from the invision pics, but they don't match the photon design system colors.